src/libs/OptionsListUtils.js

```diff
@@ -652,12 +652,10 @@ function getCategoryOptionTree(options, isOneLine = false) {
 /**
  * Build the section list for categories
  *
- * @param {Object[]} categories
- * @param {String} categories[].name
- * @param {Boolean} categories[].enabled
- * @param {Object[]} recentlyUsedCategories
- * @param {String} recentlyUsedCategories[].name
- * @param {Boolean} recentlyUsedCategories[].enabled
+ * @param {Object} categories
+ * @param {String} categories.name
+ * @param {Boolean} categories.enabled
```

Comment:
I am not sure that it types properly, because we have an object of objects like:
```javascript
{
    Food: {
        name: "Food",
        enabled: true,
    }
}
```

Reply:
What do you mean? It matches that, or am I missing something?

Reply:
```javascript
// What we actually have.
{
    Food: {
        name: 'Food',
        enabled: true
    },
    Food2: {
        name: 'Food2',
        enabled: false
    }
}

// What we have in JSDoc
{
    name: 'Food',
    enabled: true,
}
```